CHANGELOG.md

```diff
@@ -10,10 +10,41 @@ Mapbox welcomes participation and contributions from everyone.
 
 # 10.5.0 May 4, 2022
 ## Features ✨ and improvements 🏁
+* Make map projection part of the style-spec and introduce new methods `StyleInterface#setProjection` / `StyleInterface#getProjection`. Setting projection supports Style DSL as well. ([#1255](https://github.com/mapbox/mapbox-maps-android/pull/1255), [#1314](https://github.com/mapbox/mapbox-maps-android/pull/1314))
+* Optimize how plugins handle settings changes. Call `applySettings` only when settings value changes. ([#1189](https://github.com/mapbox/mapbox-maps-android/pull/1189))
```

Comment:
Can we add only projection changelogs to this pr? we can add rest of the changelog later when cutting release ?

Reply:
I worry that when someone (not me) will be cutting the release - he / she might simply leave changelog entry for this PR as well as copy the one from beta. But perhaps you're correct, I'll revert it

Reply:
I will keep 👀  on the changelog at the time of release :)